### ISSUE TITLE
Update prevents ST2 from opening on Windows

### ISSUE THREAD

**Issue by twolock** [andrewheiss/SublimeStataEnhanced#35]
A few of my coworkers and I started having trouble opening Sublime Text 2 on Windows after this package was updated. If any of us try to open Sublime, we get an error message that says that it can't open a file in the Stata Enhanced AppData directory. Then it closes. We all independently fixed the problem with a fresh install before anyone thought to report the issue, so I don't have the actual error message. I might be able to get it tomorrow or so. I would guess that other people in my office are having this problem. Sorry I can't provide more details. I just wanted to make sure someone brought it up.

I am using Sublime Text 2 Version 2.0.2 on Windows 7. I have heard about this happening on Windows 10 as well.

**Comment by andrewheiss** [comment]
Hrm. This might be because the package now first looks for a settings file in the User directory instead of relying just on the package settings (since package updates would overwrite any modifications to the settings file), so ST might just be complaining about not seeing that file. A clean install seems to fix it, though. 

I just did a fresh install of the package on a Windows machine and couldn't reproduce the error, so that's my best guess…

**Comment by twolock** [comment]
Thanks for the response. Here's the error:

"Error trying to parse file: Trailing comma before closing bracket in C:\Users\USERNAME\AppData\Roaming\Sublime Text 2\Packages\Stata Enhanced\Main.sublime-menu:94:25".

I just tried reinstalling the package using Package Control, and it has started giving me the same error. I checked Main.sublime-menu and sure enough found a trailing comma in line 94. Removing it fixed the problem.
